Marquee/Cells/Collection Cell/CollectionViewCell.swift

```diff
@@ -10,8 +10,8 @@ import UIKit
 class CollectionViewCell: UICollectionViewCell {
     
     @IBOutlet weak var poster: UIImageView!
-    
-    
+    @IBOutlet weak var poster2: UIImageView!
```

Comment:
Where is `poster2` used?

Reply:
Whoops! This was leftover from a previous experiment. 

removed 👍